pkg/apis/pipeline/v1alpha1/run_validation_test.go

```diff
@@ -36,20 +36,18 @@ func TestRun_Invalid(t *testing.T) {
 		want *apis.FieldError
 	}{{
 		name: "missing spec",
-		run:  &v1alpha1.Run{},
-		want: apis.ErrMissingField("spec"),
-	}, {
-		name: "invalid metadata",
-		run: &v1alpha1.Run{
-			ObjectMeta: metav1.ObjectMeta{Name: "run.name"},
-		},
-		want: &apis.FieldError{
-			Message: "Invalid resource name: special character . must not be present",
-			Paths:   []string{"metadata.name"},
+		run:  &v1alpha1.Run{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "temp",
```

Comment:
This seems to be required if we want to use `IsDNS1123Label` for validation

Reply:
Will tackle the other 38 failing tests tomorrow

Reply:
sounds good, thanks @psschwei